packages/jss-keyframes-helper/README.md

```diff
@@ -0,0 +1,80 @@
+# jss-keyframes-helper
+
+Helper for generating keyframes with a unique id in jss
+
+## Installation
+
+> yarn add jss-keyframes-helper
```

Comment:
we had always npm in docs, should we stay consistent with this?

Reply:
Yes, just wrote yarn because I mostly use yarn